server/internal/graphql/resolver/error.go

```diff
@@ -87,15 +91,33 @@ func ErrPublishBroker(err error) error {
 	return errors.New(ErrResolverPublishBrokerCode, errors.Alert, []string{"Unable to publish to broker", err.Error()}, []string{"Unable to create a broker publisher"}, []string{"Could be a network issue", "Meshery Broker could have crashed"}, []string{"Check if Meshery Broker is reachable from Meshery Server", "Check if Meshery Broker is up and running inside the configured cluster"})
 }
 
+func ErrMakeDir(err error, dir string) error {
+	return errors.New(ErrMakeDirCode, errors.Alert, []string{"Unable to create directory/folder", dir}, []string{err.Error()}, []string{}, []string{})
+}
+
+func ErrOpenFile(file string) error {
```

Comment:
ErrOpenFile, ErrCopy, ErrDatabaseOpen. All of these are redundant and add no useful information. The error string already will have this information. Instead use a better meshkit Error which has appropriate context.